facade/disabledNodeFacade_test.go

```diff
@@ -0,0 +1,146 @@
+package facade
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDisabledNodeFacade_AllMethodes(t *testing.T) {
+	t.Parallel()
+	defer func() {
+		r := recover()
+		if r != nil {
+			assert.Fail(t, fmt.Sprintf("should have not panic: %v", r))
+		}
+	}()
+
+	dnf, _ := NewDisableNodeFacade()
+	dnf.SetSyncer(nil)
+	dnf.SetTpsBenchmark(nil)
+	r := dnf.TpsBenchmark()
+	assert.Nil(t, r)
+	dnf.StartBackgroundServices()
+	b := dnf.RestAPIServerDebugMode()
+	assert.Equal(t, b, false)
+	s1 := dnf.RestApiInterface()
+	assert.Equal(t, s1, emptStr)
```

Comment:
always use `expected` before `actual`. 
so for example, here you should use:
`assert.Equal(t, emptStr, s1)`
apply in the entire test

Reply:
done